docs/tutorials/basic/data.md

```diff
@@ -180,6 +184,30 @@ mod = mx.mod.Module(symbol=net)
 mod.fit(data_iter, num_epoch=5)
 ```
 
+A note on python 3 usage: Lot of the methods in mxnet use string for python2 and bytes for python3. 
+In order to keep this tutorial readable, we are going to define a utility function that converts
+string to bytes in python 3 environment
+
+```python
+def str_or_bytes(str):
```

Comment:
mxnet.base.py_str

Reply:
If I understand it correctly, the output of py_str is a _string_ and input can be either a _string_ or _bytes_. What we are looking for here is the other way around with input always a string but output can either be a string or bytes. 

We can optionally add a similar function to the base.py to get this kind of job done. Let me know your thoughts.